doc/DEVELOPER.md

```diff
@@ -1,40 +1,41 @@
 ## Getting started
 
-* Install Go - you can download it from [golang.org](https://golang.org/) for your architecture or use `brew install go` if you use Homebrew. We recommend 1.13 or higher
+* Install Go - you can download it from [golang.org](https://golang.org/) for your architecture or use `brew install go` if you use Homebrew. We recommend 1.13 or higher.
 * Clone this repository
 * Install `goreleaser` from [goreleser](https://github.com/goreleaser/goreleaser) and `chglog` from [chglog](https://github.com/goreleaser/chglog)
 * To compile the CLI, run `make` in the directory
 * Use the IDE of preference, we like **Visual Studio Code**
 
 ## Overall architecture
 
-Currently all the components are integrated using [Cobra](https://github.com/spf13/cobra) as the base for our CLI and Civogo which is the Go interface to our API, in addition to that we use other libraries for colours, tables and spinners. We also use `editorconfig` to make all the encoding standard regardless of the IDE used. To see the Civo API document you can go [here](https://www.civo.com/api)
+Currently, all the components are integrated using [Cobra](https://github.com/spf13/cobra) as the base for our CLI and Civogo which is the Go interface to our API, in addition to that we use other libraries for colours, tables and spinners. We also use `editorconfig` to make all the encoding standard regardless of the IDE used. To see the Civo API document you can go [here](https://www.civo.com/api).
+
 
 ### Library used in the CLI
 
 * [Cobra](https://github.com/spf13/cobra) (to build the CLI)
 * [Civogo](https://github.com/civo/civogo) (library for Civo's API)
-* [Spinner](github.com/briandowns/spinner) (to do the waiting spinners)
-* [Color](https://github.com/gookit/color) (to color the messages, it is also compatible with Windows)
+* [Spinner](https://github.com/briandowns/spinner) (to do the waiting spinners)
+* [Color](https://github.com/gookit/color) (to color the messages, also compatible with Windows)
 * [Go-homedir](https://github.com/mitchellh/go-homedir) (to get the home of the system user)
-* [Tablewriter](https://github.com/olekukonko/tablewriter) (the utility to make the output tables to the terminal)
+* [Tablewriter](https://github.com/olekukonko/tablewriter) (the utility to create and generate text based tables in the terminal)
 * [Go-update](https://github.com/tj/go-update) (it is to be able to auto-update the binary)
 
```

Comment:
You can add https://github.com/alejandrojnm/go-pluralize 👍

Reply:
will it be a good idea to add the installation commands for the libraries used?